packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -30,6 +30,19 @@ const nextConfig = {
   // Enable React Strict Mode
   reactStrictMode: true,
 
+  // use this configuration to ensure that only images from the whitelisted domains
+  // can be served from the Next.js Image Optimization API
+  // see https://nextjs.org/docs/app/api-reference/components/image#remotepatterns
+  images: {
+    remotePatterns: [
+      {
+        protocol: 'https',
+        hostname: 'edge.**',
```

Comment:
would it be better to use a more specific pattern, like "edge.*.cloud*"?

Reply:
Also, the pattern needs to work for non-prod XMC environments as well (i.e. `https://edge-<env>.sitecore-<env>.cloud`).